pkg/apis/configuration/v1/kongprotocol_types.go

```diff
@@ -1,7 +1,7 @@
 package v1
 
-// + KongProtocol is a valid Kong protocol
-// + This alias is necessary to deal with https://github.com/kubernetes-sigs/controller-tools/issues/342
+// KongProtocol is a valid Kong protocol
```

Comment:
```suggestion
// KongProtocol is a valid Kong protocol.
```

Reply:
Adding a dot here, as the comment generated in `docs/api-reference.md` lacks it.